### ISSUE TITLE
Implement the functionality to apply automatic labels and categories to the data

### ISSUE THREAD

**Issue by AKuederle** [learn-and-code/MoneyPy#10]
This is a "main" issue. This means there will likely be various smaller issues related to this one. Please link them back here.

**Comment by fabianhough** [comment]
I think this would be interesting to try and tackle!

**Comment by fabianhough** [comment]
At first glance, it looks like what we need to do is first figure out how the internal layout of the banking data needs to look like?
I'm not sure if we should decide what we want the internal banking data to look like, or if the "Import Data" team should decide based on what they'll be importing.
Let me know if this isn't what anyone else was thinking.

**Comment by AKuederle** [comment]
Correct! However, I wanted to kind of circumvent that for now, as the banking object would be something everybody would modify, which would make working on specfic components harder. For now I would assume, that the banking data pbject you have to work with is pandas dataframe.
Which fields these object will have is something that the import team and this team has to figure out.
But for know I think we can focus on things that are general, like a `FieldContains` filter. And then when we figured out the exact structure of the banking data, we can build more specialised filters.

The team that will actually implement the basis object of the banking model will be the "load and store" team. So at the end you will need to communicate. I think it is important that you decide which fields you guys want to add to the structure (e.g. a label field, a category field) and then ask the import people which fields they will provide and then forward both information to the backend team

**Comment by vanquil** [comment]
oi
